Sources/Valet/Internal/SecItem.swift

```diff
@@ -110,10 +76,9 @@ internal final class SecItem {
     
     // MARK: Internal Class Methods
     
-    internal static func copy<DesiredType>(matching query: [String : AnyHashable]) -> DataResult<DesiredType> {
-        guard query.count > 0 else {
-            ErrorHandler.assertionFailure("Must provide a query with at least one item")
-            return .error(errSecParam)
+    internal static func copy<DesiredType>(matching query: [String : AnyHashable]) throws -> DesiredType {
+        if query.isEmpty {
+            assertionFailure("Must provide a query with at least one item")
```

Comment:
We assert, and then continue here. We'll end up throwing a `couldNotAccessKeychain` later in this method. But if we hit this assertion, we have problems _within_ Valet.